src/test/java/com/yelp/nrtsearch/server/warming/WarmerTest.java

```diff
@@ -167,17 +191,47 @@ private List<SearchRequest> getTestSearchRequests() {
               .setIndexName(index)
               .setQuery(
                   Query.newBuilder()
-                      .setTermQuery(TermQuery.newBuilder().setField("field" + i).build())
-                      .build())
+                      .setFunctionScoreQuery(
+                          FunctionScoreQuery.newBuilder()
+                              .setQuery(
+                                  Query.newBuilder()
+                                      .setTermQuery(TermQuery.newBuilder().setField("field" + i)))
+                              .setScript(Script.newBuilder().setLang("js").setSource("3 * 5"))))
               .build();
       testRequests.add(searchRequest);
     }
     return testRequests;
   }
 
+  private List<SearchRequest> getTestBasicSearchRequests() {
+    List<SearchRequest> testRequests = new ArrayList<>();
+    SearchRequest searchRequest =
+        SearchRequest.newBuilder()
+            .setIndexName(index)
+            .setQuery(Query.newBuilder().setTermQuery(TermQuery.newBuilder().setField("field0")))
+            .build();
+    testRequests.add(searchRequest);
+
+    searchRequest =
```

Comment:
should we add a comment just to make it clear that we are expecting the second request to not be warmed up due to the random seed? When I quickly read the test, I expected this request to not have a script query since the name of the method was getTest`Basic`SearchRequests